sda-download/dev_utils/config.yaml

```diff
@@ -4,7 +4,6 @@ app:
   serverkey: "./dev_utils/certs/download-key.pem"
   port: "8443"
   middleware: "default"
```

Comment:
```suggestion
  middleware: "default"
  c4gh:
    passphrase: ""
    privateKeyPath:  ""
```

Reply:
not sure whys this is not marked as outdated but it's changed now